docs/zh/use_case/dolphinscheduler_task_demo.md

```diff
@@ -0,0 +1,149 @@
+# DolphinScheduler OpenMLDB Task：打造端到端MLOps工作流
+
+## 导读
+在机器学习从开发到上线的业务闭环中，数据处理、特征开发、模型训练往往要耗费大量的时间和人力。
+
+为给 AI 模型构建及应用上线提供便利，简化机器学习建模工程化的流程，我们开发了 DolphinScheduler OpenMLDB Task，将特征平台能力融入 DolphinScheduler 的工作流，链接特征工程与调度环节，打造端到端 MLOps 工作流，帮助开发者专注于业务价值的探索。
+
+本文将为大家简要介绍并实际演示 DolphinScheduler OpenMLDB Task 的操作流程。
+
+```{seealso}
+[DolphinScheduler OpenMLDB Task 官方文档](https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/guide/task/openmldb.html)
+```
+
+## 场景和功能
+### DolphinScheduler OpenMLDB Task 为什么诞生
+
+![eco](images/ecosystem.png)
+
+作为提供生产级数据及特征开发全栈解决方案的开源机器学习数据库，OpenMLDB 提升易用性、降低使用门槛的关键点来自上下游的连通。如上图所示，接入数据源可以使得 DataOps 内的数据更加简单轻松地流入 OpenMLDB，而 OpenMLDB 供给的特征也需要顺滑地进入 ModelOps，接受训练。
+
+为了减少每个开发者手动完成接入带来的巨大的工作量，提升 OpenMLDB 使用的便捷度，我们也开发了 OpenMLDB 接入 Deployment 和 Monitoring的功能。
+
+本次想和大家重点介绍的就是 OpenMLDB 接入 DolphinScheduler 工作流的框架。
+
+DolphinScheduler OpenMLDB Task 可以更简单地操作 OpenMLDB，同时 OpenMLDB 任务也被 Workflow 管理，更加自动化。
+
+### DolphinScheduler OpenMLDB Task 可以做什么
+
+OpenMLDB 希望能达成开发即上线的目标，让开发回归本质，而不是在工程化落地中耗费过多心思。
+
+通过编写 OpenMLDB Task，我们可以实现 OpenMLDB 的离线导入、特征抽取、SQL 部署上线、在线导入等需求，也可以在DolphinScheduler 中编写一个完整的使用 OpenMLDB 的训练上线流程。
+
+![task func](images/task_func.png)
+
+比如我们设想的最简易的用户操作流程，如上图所示，流程中的1-4步正对应离线导入、特征抽取、SQL 部署上线和在线导入，均可以通过 DolphinScheduler OpenMLDB Task 编写。
+
+除了OpenMLDB中的SQL上线，实时预测还需要模型上线。所以接下来，我们将基于 kaggle 比赛中的 TalkingData 广告欺诈检测场景，为大家演示如何使用 DolphinScheduler OpenMLDB Task 编排一个完整的机器学习训练上线的流程。TalkingData 比赛详情见[talkingdata-adtracking-fraud-detection](https://www.kaggle.com/competitions/talkingdata-adtracking-fraud-detection/discussion)。
+
+## 实践演示
+### 环境配置
+演示可以在 macOS 或 Linux 上运行，也可以使用我们提供的 OpenMLDB 镜像：
+```
+docker run -it 4pdosc/openmldb:0.5.1 bash
+```
+```{attention}
+DolphinScheduler 需要操作系统的用户，并且该用户需要有 sudo 权限。所以推荐在 OpenMLDB 容器内下载并启动 DolphinScheduler。否则，请准备有sudo权限的操作系统用户。
+```
+
+在容器中，可以直接运行以下命令启动 OpenMLDB cluster。
+```
+./init.sh
```

Comment:
cd talkingdata/

Reply:
init.sh is in /work, not /work/talkingdata